### PR TITLE
Preparing release 2.0.0

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

**Comment by leplatrem** [comment]
I suggest that we don't copy paste the changelog, but we just mention the dependencies version changes with a link to the related release pages.

We could manage what is really relevant for distribution builders though: eg. the part of the breaking changes that impact configuration files or commands.

File: setup.py [hunk 1]
```diff
@@ -12,22 +12,24 @@ def read_file(filename):
...
-    "kinto-signer",
-    "cliquet-fxa",
-    "boto",
+    "cliquet[monitoring,postgresql]>=3.1,<4",
+    "kinto>=2.0,<3.0",
+    "kinto-attachment>=0.4,<1.0",
+    "kinto-changes>=0.1,<1.0",
+    "kinto-signer>=0.1,<1.0",
+    "cliquet-fxa>=1.4,<2.0",
+    "boto>=2.39,<3.0",
```
Comment (leplatrem) [inline setup.py]: no changes in `requirements.txt` ?

File: setup.py [hunk 2]
```diff
@@ -12,22 +12,24 @@ def read_file(filename):
...
 ]
 ENTRY_POINTS = {}
 DEPENDENCY_LINKS = []
 
 setup(name='kinto-dist',
-      version='0.1.0',
+      version='2.0.0',
```
Comment (leplatrem) [inline setup.py]: From 0 to 2 ?

Comment (Natim) [inline setup.py]: I was planning to follow kinto releases but I might be a wrong idea.

File: setup.py (hunk 1 above)
Comment (Natim) [inline setup.py]: The requirements.txt is already up-to-date.

**Pull request merged**